### PR TITLE
🔊 Add Verbose Logging for Debugging (Temporary)

### PR DESCRIPTION
**Pull request by mistryrn**
### Consent UI
* Set NextAuth `debug` to `true`
* Add request logging in Next.js middleware
* Format debug logging on Home and Dashboard

## PR Readiness Checklist

- [ ] "Expected Outcome(s)" in ticket have been met
- [ ] Ticket number included in PR title
- [ ] Connected ticket to PR
- [ ] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [ ] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [ ] New environment variables added to `.env.schema` files, `README.md`
- [ ] Added copyrights to new files

### CODE REVIEW

File: apps/consent-ui/src/middleware.ts [hunk 1]
```diff
@@ -32,6 +32,12 @@ export const config = {
 };
 
 function middleware(request: NextRequest) {
+	// TODO: for debugging, TO BE REMOVED
+	const requestUrl = request.url;
+	const requestHeaders = request.headers;
+	console.info('[middleware] Request URL:', requestUrl);
+	console.info('[middleware] Request Headers:', requestHeaders);
```
Comment (mistryrn) [inline apps/consent-ui/src/middleware.ts:39]: Technically this doubles the Header logging on the Homepage and Dashboard, but from my testing this middleware fires first before the route/page.tsx does so _just in case_ there's something going on between middleware to page/route render I figured it couldn't hurt to log the headers here too.

This Headers object also has a few things the `next/headers` one doesn't.

File: apps/consent-ui/src/app/authConfig.ts [hunk 2]
```diff
@@ -147,4 +147,6 @@ export const authConfig = {
 			return await doFinalSignoutHandshake(token);
 		},
 	},
+	// TODO: for debugging, TO BE REMOVED
+	debug: true,
```
[the remark this answers is not in the document]
Comment (anncatton) [reply apps/consent-ui/src/app/authConfig.ts:151]: ou nice!

File: apps/consent-ui/src/app/[lang]/page.tsx [hunk 3]
```diff
@@ -25,6 +25,6 @@ import HomeComponent from 'src/components/views/Home';
 export default async function Home({ params: { lang } }: { params: { lang: ValidLanguage } }) {
 	// TODO: for debugging, TO BE REMOVED
 	const headersList = headers();
-	console.info(`HEADERS FROM ROUTE - HOME - `, JSON.stringify(headersList));
+	console.info(`[page.tsx - Home Route] Headers:`, JSON.stringify(headersList, null, 2));
```
Comment (anncatton) [inline apps/consent-ui/src/app/[lang]/page.tsx:28]: oh thanks for catching my missed formatting!

File: apps/consent-ui/src/middleware.ts (hunk 1 above)
Comment (anncatton) [reply apps/consent-ui/src/middleware.ts:39]: it's weird, i wasn't seeing any header info at all in the middleware, i must have missed something in my log. thanks for adding!

**Review by anncatton — APPROVED** [review]

**Pull request merged**